command/pr.go

```diff
@@ -249,7 +246,7 @@ func prList(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		fmt.Fprintf(cmd.ErrOrStderr(), "Opening %s in your browser.\n", openURL)
+		fmt.Fprintf(cmd.ErrOrStderr(), "Opening %s in your browser.\n", displayURL(openURL))
```

Comment:
Oh. I purposely did not use the `displayURL` function, thinking that it would be better to show the perfect URL, but okay :+1:

Reply:
The URL-encoded query would be hard to read, so I've made a call to not display it. We also avoid printing query strings in other feedback messages.